src/rules/csslint.js

```diff
@@ -1,32 +0,0 @@
-import { CSSLint } from 'csslint';
```

Comment:
Do we have a replacement for this?
Or if we want a replacement in the near future, do we have an issue addressing this?

Reply:
I believe it's better to have a replacement, what do you think?

Reply:
Referencing issue: https://github.com/htmlhint/HTMLHint/issues/213